src/code-block/settings.ts

```diff
@@ -0,0 +1,62 @@
+import { App, MarkdownPostProcessorContext, normalizePath, parseYaml } from "obsidian";
+import renderError from "~/code-block/errors";
+
+export interface Settings {
+  path: string;
+  limit: number;
+  recursive: boolean;
+  sort: "asc" | "desc";
+  sortby: "name" | "mtime" | "ctime";
+}
+
+const DEFAULT_SETTINGS: Settings = {
+  path: "",
+  limit: 0,
+  recursive: true,
+  sort: "desc",
+  sortby: "mtime",
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type AnyObject = { [key: string]: any };
```

Comment:
[🔧](https://github.com/erikthedeveloper/code-review-emoji-guide#:~:text=:wrench:) Use `unknown` rather than `any` so you don't have the need to disable eslint on that one

Reply:
Actually ended up being `type AnyObject = { [key: string]: AnyObject }`